### PR TITLE
Fixes AIs being able to see cult runes

### LINKED ISSUE #17426: AI can see cult runes
https://github.com/tgstation/-tg-station/blob/35bfb9abb6d3e7952d14a1781c9ffc3a68d2df00/code/modules/mob/living/silicon/ai/login.dm
AI is supposed to just see blood splatters, but it can see runes.

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #17426

:cl: Cyberboss
fix: AIs can no longer see cult runes properly
/:cl:

They now see blood spatter

### CODE REVIEW

**Comment by jughu** [comment]
Cult grows stronger and stronger... every day...

**Comment by JJRcop** [comment]
@jughu it used to be this way but was broken somehow

**Comment by Anonmare** [comment]
I kinda wish x-ray vision was the same way

**Comment by Supermichael777** [comment]
but why?  If your dumb enough to rune up somewhere an ai can see you really deserve it.

File: code/game/gamemodes/cult/runes.dm [hunk 1]
```diff
@@ -38,11 +39,30 @@ To draw a rune, use an arcane tome.
...
 	var/keyword //The actual keyword for the rune
+	var/image/ai_image
 
 /obj/effect/rune/New(loc, set_keyword)
 	..()
 	if(set_keyword)
 		keyword = set_keyword
+	runes += src
+
+	var/obj/effect/decal/cleanable/blood/splatter/s = new(loc)
```
Comment (RemieRichards) [inline code/game/gamemodes/cult/runes.dm]: ```var/obj/effect/decal/cleanable/blood/splatter/s = /obj/effect/decal/cleanable/blood/splatter```

File: code/game/gamemodes/cult/runes.dm [hunk 2]
```diff
@@ -38,11 +39,30 @@ To draw a rune, use an arcane tome.
...
+	ai_image = image(s.icon, icon_state = s.icon_state, loc = src.loc)
```
Comment (RemieRichards) [inline code/game/gamemodes/cult/runes.dm]: ```ai_image = image(initial(s.icon), icon_state = initial(s.icon_state), loc = src.loc)```

File: code/game/gamemodes/cult/runes.dm [hunk 3]
```diff
@@ -38,11 +39,30 @@ To draw a rune, use an arcane tome.
...
+	qdel(s)
```
Comment (RemieRichards) [inline code/game/gamemodes/cult/runes.dm]: remove

File: code/game/gamemodes/cult/runes.dm [hunk 4]
```diff
@@ -38,11 +39,30 @@ To draw a rune, use an arcane tome.
 
 	var/req_keyword = 0 //If the rune requires a keyword - go figure amirite
 	var/keyword //The actual keyword for the rune
+	var/image/ai_image
```
Comment (RemieRichards) [inline code/game/gamemodes/cult/runes.dm]: make this ```/static```, so they all reuse the same image, 1 image many runes.
(also check below, ```if(!ai_image) ... do image shit...```)

**Review by RemieRichards — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/cult/runes.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/game/gamemodes/cult/runes.dm]: I don't think you even need the ` = /obj/effect/decal/cleanable/blood/splatter` part, do you?

Comment (RemieRichards) [inline code/game/gamemodes/cult/runes.dm]: you do in 90% of cases, the case where it's skippable is where it's a static variable.

**Comment by Cyberboss** [comment]
@RemieRichards The reasoning I have for not having it static is that so it can be a different splatter icon for each rune

**Comment by Cyberboss** [comment]
Hiding from the fruit.

**Comment by coiax** [comment]
@Cyberboss make runes that have been disguised as crayon runes visible to the AI. Otherwise that's a tell that the crayon runes are real.

**Comment by Cyberboss** [comment]
Crayon runes aren't related to real runes at all so I think we're ok

**Comment by coiax** [comment]
@Cyberboss you misunderstand, cultists have a talisman that changes the description of their runes to make them the same as crayon runes, meaning that non-cultists can't tell the difference. But they're still cult runes.

**Comment by Cyberboss** [comment]
Crayon runes are now invisible as well to prevent powergamers from asking the AI what it looks like

**Comment by Cyberboss** [comment]
Just tested it out and apparently this image method doesn't apply the alpha channel for some reason

untitled

What do?

**Comment by coiax** [comment]
That's not quite what I meant

**Comment by coiax** [comment]
But okay

**Comment by jughu** [comment]
Lets buff the most easiest gamemode even more 👍

**Comment by optimumtact** [comment]
@RemieRichards please update your review

**Review by RemieRichards — APPROVED** [review]

**Pull request merged**